packages/hardhat-core/src/internal/core/config/config-loading.ts

```diff
@@ -41,7 +44,7 @@ export function loadConfigAndTasks(
     hardhatArguments !== undefined ? hardhatArguments.config : undefined;
 
   configPath = resolveConfigPath(configPath);
-
+  log(`Hardhat config path located at ${configPath}`);
```

Comment:
```suggestion
  log(`Loading Hardhat config from ${configPath}`);
```

What about this message instead? It makes it clear what was happening at the time the log was printed.

Reply:
Yeap. Clearer